.eslintrc.json

```diff
@@ -26,11 +26,6 @@
       "extends": ["plugin:@nx/typescript"],
       "rules": {}
     },
-    {
```

Comment:
Why remove this? I don't think there's a downside to keeping the Nx preset for `.js` files (it'll only run if the glob matches anyway).

Reply:
added .. guess it was due to some issues